docs/components/plugins/APISection.tsx

```diff
@@ -17,6 +17,7 @@ const LATEST_VERSION = `v${require('~/package.json').version}`;
 type Props = {
   packageName: string;
   apiName?: string;
+  test_forceVersion?: string;
```

Comment:
probably not necessary to prefix this with `test_` -- there is likely not a good use case for using this in other contexts but we can just call it `version`

Reply:
`version` is reserved in the component for context version, so I just drop the `test_` prefix, to avoid additional renames.